src/Components/CommonParts/Container.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import Container from 'react-bootstrap/Container';
+import BreadCrumb from '../DefaultLayout/DefaultBreadcrumb';
+
+export interface Props {
+  children: ReactNode,
+  pageTitle: string
+}
+
+const ContainerComponent: FunctionComponent<Props> = props => {
+  return (
+    <main className="main">
+      <nav>
+        <BreadCrumb pageTitle={props.pageTitle}/>
+      </nav>
+      <Container fluid>
+          {props.children}
+      </Container>
+    </main>
+  );
+}
+
+export default ContainerComponent;
```

Comment:
As this is part of CoreUI layout, should move into Layout component folder and export via index.ts